src/main/resources/config.yml

```diff
@@ -37,6 +37,9 @@ AvailableLanguages:
 - "english"
 DefaultLanguage: "english"
 TeamChatPrefix: "t "
+# Get signature and texture from here: https://mineskin.org/gallery?query=Camera
```

Comment:
Remove comment, but it's just a suggestion. Up to you

Reply:
I had to do a lot of research to know how to work with skins, I think it's really useful to have that there so you can understand fast how do change the skin.

Reply:
Fair :)